README.md

```diff
@@ -18,6 +23,13 @@ Open a notepad in the Firefox toolbar to store notes about your browsing.
 | `npm run build` | Builds the application as a Web Extension.
 | `npm start`     | Launches Firefox with the Web Extension.
 
+NOTE: if you are running on Windows, and experience a build error use `$ npm build`
```

Comment:
`npm run buid` is giving an error 
![screenshot 201](https://user-images.githubusercontent.com/21216969/29278324-0388b36a-8132-11e7-94c6-23170d10b18a.png)
`npm build` is working fine on windows 
![screenshot 203](https://user-images.githubusercontent.com/21216969/29278446-67a50632-8132-11e7-938e-6674f4817e2f.png)